### PR TITLE
Extra material

### PR DESCRIPTION
**Pull request by fasterius**
Added some extra material for Git and Conda involving decorating the prompt and some potentially useful aliases, for those that like that sort of thing.

### CODE REVIEW

File: docs/conda.md [hunk 1]
```diff
@@ -241,3 +241,49 @@ Note that all that was needed to carry out the analysis and generate these files
...
+
+By default, Conda adds the name of the currently activated environment to the
+end of your command line prompt. This is a good thing, as it makes it easier to
+keep track of what environment and packages you have access to. The way this is
+done in the default implementation becomes an issue when using absolute paths
+for environments (specifying `conda env create -p <path/to/environment>`,
+though, as the entire path will be added to the prompt. This can take up a lot
+of unnecessary space, but can be solved in a number of ways. Here we present
+one of them, using a custom bash function:
+
```
[the remark this answers is not in the document]
Comment (fasterius) [reply docs/conda.md]: I didn't even know you could do that! I wonder when they added that in... Good idea, I'll add it in!

**Comment by fasterius** [comment]
Good points! My thoughts:

1. I don't if it would be suitable to have that in the NGS course since they're not using git or Conda (as far as I remember), and the aliases here are specifically for git/Conda.
2. Yes, but they are at the end of the lesson, labeled as extra material. Do you still think they might get confused by this? My thinking was that only the students that come all the way to the end of the lesson would read these, and only apply them if they feel they want to.

Another thought is that while the bash profile is not really a part of either git or Conda, I don't see any other NBIS course available in our current catalogue where it would be more suitable than here. Whether that constitutes a good enough reason to include it is another matter.

**Comment by johnne** [comment]
Hmm yes maybe it could make sense to have it at the end. While I agree both git and conda could benefit from more content like this (and for instance include a dedicated section on conda config and tips and tricks for improving the performance of conda) I think I have to read through it to see how it flows.

What are your thoughts @verku?

**Comment by verku** [comment]
Hi, I think that the material can definitely be useful to some course participants! My major thought when reading the instructions was that we might want to change the description of the material. In the current version, it reads to me as if everyone should actually apply these changes and adjust their settings, but I would rather see it as tips and tricks from more experienced git and conda users. We could flag this material for example as suggestions how to make your coding experience easier if you are using git and/or conda in your daily life, not as extra material to work through. For someone who is more inexperienced with the command line and who uses git or conda very rarely, the aliases could otherwise become quite confusing.

As extra material I would suggest to include other topics that could be more of use even for irregular users (I put some ideas into the project).

What do you think?

**Comment by fasterius** [comment]
Sounds like a good idea to change the text to reflect that it's not extra material to be worked through, rather tips & tricks! I can fix that. Where did you add your ideas, @verku ?

**Comment by verku** [comment]
Great, thanks!

John created two items in the project under "New Ideas", one for conda and one for git. My suggestions are in the two lists within these items.

**Comment by johnne** [comment]
I made some minor updates for clarity (in my view ;)). For my part I'm happy to merge this PR now. But I guess we can also keep working on the branch according to suggestions in issues #19 and #20. What do you guys think?

**Comment by fasterius** [comment]
Okay, let's I'll merge it and then we can continue working on the branch for new content as well!

**Comment by verku** [comment]
It sounds good to me to work on the suggestions in issues #19 and #20!

**Pull request merged**